src/Core/Settings.cpp

```diff
@@ -1539,6 +1539,28 @@ Possible values:
 
 - 0 — Disabled.
 - 1 — Enabled.
+)", 0) \
+    DECLARE(Bool, use_skip_indexes_for_top_n, false, R"(
```

Comment:
just to have a consistent wording in the codebase: `_top_k` and `TopK filtering`?